tools/modules/system/module_armbian_firmware.sh

```
@@ -251,9 +254,9 @@ function module_armbian_firmware() {
 			${module_options["module_armbian_firmware,feature"]} ${commands[2]} "" "" hide
 
 			# release Armbian packages from hold
-			apt-mark unhold ${packages[@]} >/dev/null 2>&1
+			apt-mark unhold ${packages[@]} > /dev/null 2>&1
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)